_install-and-configure/install-dashboards/tls.md

```diff
@@ -19,7 +19,7 @@ Setting | Description
 `server.ssl.certificate` | If `server.ssl.enabled` is set to `true`, specifies the full path to a valid Privacy Enhanced Mail (PEM) server certificate for OpenSearch Dashboards. You can [generate your own certificate]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority (CA).
 `server.ssl.key` | If `server.ssl.enabled` is set to `true`, specifies the full path to the key for your server certificate, for example, `/usr/share/opensearch-dashboards-1.0.0/config/my-client-cert-key.pem`. You can [generate your own certificate]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a CA.
 `server.ssl.keyPassphrase` | Sets the password for the key. Omit this setting if the key has no password. Optional.
-`server.ssl.keystore.path` | Uses a JKS (Java KeyStore) or PKCS12/PFX (Public-Key Cryptography Standards) file instead of a PEM certificate and key.
+`server.ssl.keystore.path` | Uses a JKS (Java keystore) or PKCS12/PFX (Public-Key Cryptography Standards) file instead of a PEM certificate and key.
```

Comment:
```suggestion
`server.ssl.keystore.path` | Uses a JKS (Java KeyStore) or PKCS12/PFX (Public-Key Cryptography Standards) file instead of a PEM certificate and key.
```